src/09-workorders/work-orders.apib

```diff
@@ -1,5 +1,74 @@
 ## Work Orders [/workOrders]
 
+### workOrders.list [GET /workOrders.list]
+
+Get a list of work orders.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + filter (object, optional)
```

Comment:
B: filtering is not in scope of the first story. There's a separate story to be picked up for that later.

Reply:
Removed.